src/components/elements/annotationSidePanel/components/annotationWizard/AnnotationWizard.tsx

```diff
@@ -0,0 +1,66 @@
+/* Import Dependencies */
+import { useState } from 'react';
+import { Row, Col } from 'react-bootstrap';
+
+/* Import Types */
+import { Dict, ProgressDot } from 'app/Types';
+
+/* Import Components */
+import { AnnotationTargetStep, AnnotationFormStep, AnnotationInstanceSelectStep } from './AnnotationWizardComponents';
+import { ProgressDots, Tabs } from 'components/elements/customUI/CustomUI';
+
+
+/* Props Type */
+type Props = {
+    schema: Dict
+};
+
+
+/**
+ * Component that renders the annotation wizard for adding annotations
+ * @returns JSX Component
+ */
+const AnnotationWizard = (props: Props) => {
+    const { schema } = props;
+
+    /* Base variables */
+    const [selectedTabIndex, setSelectedTabIndex] = useState<number>(0);
+    const tabs: { [name: string]: JSX.Element } = {
+        annotationTarget: <AnnotationTargetStep schema={schema}
+        />,
+        annotationSelectInstance: <AnnotationInstanceSelectStep />,
+        annotationForm: <AnnotationFormStep />
+    };
+    const progressDots: ProgressDot[] = [];
+
+    /* Construct progress dots */
+    Object.keys(tabs).forEach((tab, index) => {
+        progressDots.push({
+            label: tab,
+            // state: index < selectedTabIndex ? 'checked' : index === selectedTabIndex ? 'current' : 'open',
+            OnClick: () => index < selectedTabIndex && setSelectedTabIndex(index)
```

Comment:
this only goes backwards? maybe users want to use the dots to go forwards

Reply:
Well you should not be able to move forward if a step that comes before is not completed yet.
But yeah if a future step has been completed you should be able to move to that one indefinitely.
Wanna look at how to regulate this in the next PR (after the completion of the progress dots).